### DISCUSSION TITLE
Libreoffice Time format code

### DISCUSSION THREAD

**Discussion by topherbuckley** [eprbell/dali-rp2#168] [Q&A] [answered]
Not directly related to dali, but I'm trying to format my csv files for input into the manual plugin, and everytime I save the file, the time format gets reverted. I use:

`YYYY-MM-DD HH:MM:SS+00:00` 

and it looks fine, save it, close it, and reopen only to find all timestamps get formatted to 

`YYYY-MM-DD HH:MM:SS.0000"+"`

Anyone deal with this before and know of a way to ensure the formatting stays upon closing? I don't want to have to reformat all my manual files everytime I need to export. :(

**Comment by topherbuckley** [comment, marked as answer]
I think I may have found something that works. 

`YYYY-MM-DD HH:MM:SS"+00:00"`

In this way it seems libreoffice treats the last part as a string, and maybe other versions were trying to replace or interpret things outside of quotation marks as something else..but failed?

**Reply by jayr0d** [reply]
All of my imports are manual.  I start with a CSV file from the exchange.  I then import that CSV file into LibreOffice, and in the Text Import dialog, uncheck Tab and Semicolon from the Separator Options, and select all Fields and set Column type to Text.  I then manipulate the ODS file as needed, save it, and then save as CSV.

This should prevent LibreOffice from doing any sort of reformatting of your data, including dates and more importantly rounding of values.
